### ISSUE TITLE
Loops in RakuScript

### ISSUE THREAD

**Issue by Jeremi360** [rakugoteam/Rakugo-Dialogue-System#64]
This continuation of discussion about loops that we started in #53 :
https://github.com/rakugoteam/Rakugo/issues/53#issue-1202108179

I think we should support 2 types of loops:
```nim
# interation on collection
for item in items:
  item.do_something()
  
# iteration on int, inspired by nim-lang:
4 times:
  say "This will be said up 4 times."
```

**Comment by theludovyc** [comment]
We already have loops

```
loop :
# do things
jump loop
```
But yeah, without conditions we never go out of it ^^. I will work on it. I just needed to rework save system firt to use variables easily :)

**Comment by Jeremi360** [comment]
You're right, that's probably enough and it's simple. Sorry often such obvious solutions get missed.
I recently learned that C# has `goto` :open_mouth:

**Comment by theludovyc** [comment]
You need to play game like shenzenI/O (https://store.steampowered.com/app/504210/SHENZHEN_IO/) or Human Resource Machine (https://store.steampowered.com/app/375820/Human_Resource_Machine/) 😊

**Comment by Jeremi360** [comment]
I will play Human Resources then :smile:

**Comment by theludovyc** [comment]
Fixed when #75 and #81 will be closed
